frontend/apps/cloudstorage/webdav.lua

```diff
@@ -48,6 +49,36 @@ function WebDav:downloadFile(item, address, username, password, local_path, call
     end
 end
 
+function WebDav:uploadFile(url, address, username, password, local_path, callback_close)
+    local path = WebDavApi:getJoinedPath(address, url)
+    path = WebDavApi:getJoinedPath(path, ffiutil.basename(local_path))
+    local code_response = WebDavApi:uploadFile(path, username, password, local_path)
+    if code_response >= 200 and code_response < 300 then
+        UIManager:show(InfoMessage:new{
+            text = T(_("File upload succeeded:\n%1"), BD.filepath(address)),
```

Comment:
We could synchronize the wordings to avoid extra translation
https://github.com/koreader/koreader/blob/cf643c453bcbbef0d7986c625112d708558d5471/frontend/apps/cloudstorage/dropbox.lua#L73

Reply:
Done.